relayer/crates/starknet-chain-components/src/impls/send_message.rs

```diff
@@ -58,7 +58,18 @@ where
 
         let execution = account.execute_v3(calls).nonce(*nonce);
 
+        let fee_estimation = execution.estimate_fee().await.map_err(Chain::raise_error)?;
+
+        // starknet v0.13.4 requires all fee bound present.
+        let l1_gas = core::cmp::max(1, fee_estimation.l1_gas_consumed.try_into().unwrap());
```

Comment:
Let's handle error here with `.map_err(|_| Chain::raise_error("failed to convert felt to u64"))?`, and add `CanRaiseError<&'static str>` to the constraint.

It is so weird that starknet-rs makes the `TryFrom` instance raise `PrimitiveFromFeltError`, but the type is private and not exported.

Reply:
8ebd61d